eqcorrscan/core/template_gen.py

```diff
@@ -72,7 +70,7 @@ def __str__(self):
 
 def template_gen(method, lowcut, highcut, samp_rate, filt_order,
                  length, prepick, swin, process_len=86400,
-                 all_horiz=False, delayed=True, plot=False,
+                 all_horiz=False, delayed=True, plot=False, plotdir='.',
```

Comment:
Can we do what you did before and use a default plotdir=None, and if not plotdir then show please?  ~~This will maintain the same behaviour for other users without them having to work out what change has happened.~~
Sorry - lots of inconsistency in the code!  Can I get you to take the opportunity to make the default behaviour to `show=True` if plotdir is None please.  Maybe we can start working towards making the codebase consistent!

Reply:
Yes, of course.
I will work on it.
But it may take some times.
Then i will work on them, fix merging problem, and pull request later, as soon as i can.
i'm going to sleep. because It's night here right now :)
have a good day for you :)